core-bundle/src/Resources/contao/templates/image/image.html5

```diff
@@ -14,3 +14,7 @@
     <figcaption class="caption"><?= $this->caption ?></figcaption>
   <?php endif; ?>
 </figure>
+
+<?php
+
+$this->jsonLd($this->figure->getJsonLd());
```

Comment:
This would probably fail if the image template is used outside the figure builder?

Reply:
`$this->figure` always exists and yes, if you do not have it, you cannot use it.

Reply:
You mean you want to wrap it into an `if ($this->figure)`?

Reply:
> $this->figure always exists

This is a rather recent addition, though. It might fail for people that use `$this->insert('image')` from legacy contexts, so I'd rather check for it as well.

Reply:
Added in d273e2a48412e77de9de705db7b1b0f5d6f8f26c.